src/AutoUpdater.ts

```diff
@@ -28,7 +29,8 @@ export default class AppUpdater {
     }
   }
 
-  restart() {
+  async restart() {
```

Comment:
Seems odd that a restart only calls quit and install.  Maybe consider adding a comment here.

Reply:
Yea, it could be called quitAndInstall()... but I just went for the shorter name cause that's what the button in the UI says. And it does restart.